ui-tests/src/test/java/org/openmrs/reference/page/ManageRolesPage.java

```diff
@@ -0,0 +1,27 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class ManageRolesPage  extends Page {
+	private static final By ADD_ROLE = By.cssSelector("#content > a");
```

Comment:
This might be less fragile...

```suggestion
	private static final By ADD_ROLE = By.cssSelector("a[href='role.form']");
```

Reply:
Resolved thanks